frontend/src/components/Charts/DepthChart/index.tsx

```diff
@@ -0,0 +1,320 @@
+    history.push('/order/' + point.data?.order?.id);
+  }
+  
+  return bookLoading || !center || enrichedOrders.length < 1 ? (
+    <div style={{display: "flex", justifyContent: "center", paddingTop: 200, height: 420 }}>
+      <CircularProgress />
+    </div>
+  ) : (
+    <Grid container style={{ paddingTop: 15 }}>
+      <Grid 
+        container 
+        direction="row"
+        justifyContent="space-around"
+        alignItems="flex-start"
+        style={{ position: "absolute" }}
+      >
+        <Grid container xs={12} justifyContent="flex-start" alignItems="flex-start" style={{ paddingLeft: 20 }}>
+          <Select
+            value={xType}
+            onChange={(e) => setXType(e.target.value)}
+          > 
+            <MenuItem value={"premium"}>
+              <div style={{display:'flex',alignItems:'center', flexWrap:'wrap'}}>
+                {t("Premium")}
+              </div>
+            </MenuItem>
+            <MenuItem value={"base_amount"}>
+              <div style={{display:'flex',alignItems:'center', flexWrap:'wrap'}}>
+                {t("Price")}
+              </div>
+            </MenuItem>
+          </Select>
+        </Grid>
+      </Grid>
+      <Grid 
+        container 
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Grid container justifyContent="center" alignItems="center">
+          <Grid item>
+            <IconButton onClick={() => setXRange(xRange + rangeSteps)}>
+              <RemoveCircleOutline />
+            </IconButton>
+          </Grid>
+          <Grid item>
+            <Box justifyContent="center">
+              {xType === 'base_amount' ? `${center} ${currencyDict[currencyCode]}` : `${center}%`}
+            </Box>
+          </Grid>
+          <Grid item>
+            <IconButton onClick={() => setXRange(xRange - rangeSteps)} disabled={xRange <= 1}>
+              <AddCircleOutline />
+            </IconButton>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid container style={{ height: 357, padding: 15 }}>
+        <ResponsiveLine 
+          data={series} 
+          enableArea={true}
+          enablePoints={false}
+          enableGridX={false}
+          enableGridY={false}
+          useMesh={true}
+          animate={false}
+          crosshairType="cross"
+          tooltip={generateTooltip}
+          onClick={handleOnClick}
+          axisRight={{
+            tickSize: 5,
+            format: formatAxisY
+          }}
+          axisLeft={{
+            tickSize: 5,
+            format: formatAxisY
+          }}
+          axisBottom={{
+            tickSize: 5,
+            format: formatAxisX
+          }}
+          margin={{ left: 65, right: 60, bottom: 25, top: 10 }}
+          xFormat={(value) => Number(value).toFixed(0)}
+          lineWidth={3}
+          theme={getNivoScheme(theme)}
+          colors={['rgb(136, 252, 102)', 'rgb(255, 108, 57)']}
```

Comment:
While orange and green is widely used for depth charts, we could give it a touch by sticking to the RoboSats color scheme. Throughout the UI we are loosely sticking to using purple for "things you click if you want to sell" and blue for "things you click on if you want to buy".

What do you think of this?
``` javascript
          colors={[theme.palette.secondary.main,theme.palette.primary.main]}
```
These are the color references for the default MUI palette https://mui.com/material-ui/customization/palette/
When using these theme variables, the line colors will be brighter when the dark mode is enabled.